lib/util/support.js

```diff
@@ -152,4 +167,9 @@ function isSupported() {
     && (!isMobile() || supportedMobileBrowsers.includes(browser));
 }
 
-module.exports = isSupported;
+module.exports = {
+  isSupported,
```

Comment:
Since we're updating how we're exporting this, we should update other places. Can you grep the codebase and update where we are using this? I don't memorize them all but I think it includes the following:
- lib/index.js
- some integration tests
- umd tests
- serversiderender tests

Reply:
@charliesantos Do you think it's worthwhile to do this since I'd have to change it back again in the next PR? Or should I just make a new `browserdetection.js` file in this PR and we wouldn't have to adjust all the places that are importing `isSupported`?

Reply:
@PikaJoyce I'll leave it up to you. As long as we don't have a broken build when you cut an RC 😄